### PR TITLE
Fixed issue with Config.duplicate. Added test.

### PR DESCRIPTION
**Pull request by huard**

### CODE REVIEW

**Comment by Zeitsperre** [comment]
Is this going to be another patch release?

**Comment by huard** [comment]
It resolves an issue in a RavenPy notebook, but I've already supplied a fix for the notebook that works with earlier releases, so I believe not.

File: HISTORY.rst [hunk 1]
```diff
@@ -7,8 +7,9 @@ History
 * Upgraded `owslib` to `>=0.29.1`. (PR #358)
 * All operations that open NetCDF files or DAP links accept an `engine` argument. The default for all of these is `h5netcdf`. (PR #358)
 * Added `pydap` as an alternate backend for opening DAP links. (PR #358)
-* Fixed buggy CustomOutput command. (PR #360)
-* Make sure config and output paths are absolute. (PR #360)
+* Make sure config and output paths are absolute (PR #360)
+* Fixed bug in `CustomOutput` command dating from the switch to Pydantic V2 in 0.13 (PR #360)
+* Fixed bug in `Config.duplicate` dating from the switch to Pydantic V2 in 0.13 (PR #367)
```
Comment (Zeitsperre) [inline HISTORY.rst:12]: These changes need to be moved above.

```suggestion
* Fixed buggy CustomOutput command. (PR #360)
* Make sure config and output paths are absolute. (PR #360)
```

**Review by Zeitsperre — APPROVED** [review]
LGTM

**Pull request merged**